Modules/Microsoft365DSC/DSCResources/MSFT_TeamsGroupPolicyAssignment/MSFT_TeamsGroupPolicyAssignment.psm1

```diff
@@ -0,0 +1,433 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Id,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $GroupDisplayname,
+
+        [Parameter()]
+        [System.String]
+        $GroupId,
+
+        [Parameter()]
+        [System.String]
+        $PolicyType,
```

Comment:
The PolicyType parameter has a specific set of possible values, as you can see here: https://learn.microsoft.com/en-us/powershell/module/teams/new-csgrouppolicyassignment?view=teams-ps#-policytype

Please make sure you use a ValidateSet, like is also done for the Ensure parameter, to make sure just this list of values can be provided.